lib/adapters/show-document-adapter.ts

```diff
@@ -0,0 +1,43 @@
+import * as url from 'url';
+import { shell } from 'electron';
```

Comment:
@aminya This is the line that I asked you in Discord about

Reply:
Adding electron.d.ts fixes this
```ts
// Electron types
declare module 'electron' {
  export const shell: {
    openExternal(url: string): void
  }
}


```